VirtoCommerce.MarketingModule.Data/Model/CouponEntity.cs

```diff
@@ -16,6 +16,7 @@ public CouponEntity()
         [Index("IX_CodeAndPromotionId", IsUnique = true, Order = 1)]
         public string Code { get; set; }
 
+        [Range(0, Int32.MaxValue)]
```

Comment:
Need to remove this attribute, because it causes the DB schema changes and requires migration recreation